src/pages/index.js

```diff
@@ -0,0 +1,363 @@
+import {
+  enableValidation,
+  validationConfig,
+  resetValidation,
+} from "../scripts/validation.js";
+import "../pages/index.css";
+import Api from "../utils/Api.js";
+import { setButtonText } from "../utils/helpers.js";
+import { disableButton } from "../scripts/validation.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c1da3710-50e8-4d77-9af1-7ab507c7d59c",
+    "Content-Type": "application/json",
+  },
+});
+
+//destrucure the second item in the callback of the .then()
+api
+  .getAppInfo()
+  .then(([userInfo, cards]) => {
+    console.log(cards);
+
+    // render card
+    cards.forEach((card) => renderCard(card, "append"));
+    // initialCards.forEach((item) => renderCard(item, "append"));
+
+    const avatarElement = document.querySelector(".profile__avatar");
+    const nameElement = document.querySelector(".profile__name");
+    const aboutElement = document.querySelector(".profile__about");
```

Comment:
*  When I reload the page, the **old** `job`  is displayed again on the page

I think it's not correct here


You should use `profileDescription` which has already been found in line 49